styleguide/_themes/global/scss/_swatch.scss

```diff
@@ -3,6 +3,7 @@ $images: "https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.
 
 //Colors
 $white: #fff;
+$footer-gray: #b8b8b8;
```

Comment:
This is totally my opinion, and may be a question for @bighappyface, so don't necessarily act on it.  I'm just putting it out here for discussion. 

If we're gonna have region-specific colors, does it make sense to define them outside of that region (e.g. not in `_footer.scss` file in this case)? 

If so, that's fine, but perhaps _swatch.scss should be organized in such a manner to indicate that there is a master list of colors to choose from and then there are region specific colors, that reference those.  

So for example: 

```
$medium-light-gray: #b2b2b2; 
```

and then towards the end of the file: 
```
// Footer colors
$footer-color-link-hover: $medium-light-gray;
```

Otherwise, I think we may run the risk of redefining colors, or using inconsistent colors across various parts of the website. But once again, this is just my opinion so by all means provide additional feedback.